frameworks/CSharp/beetlex/Benchmarks/Benchmarks.csproj

```diff
@@ -7,9 +7,18 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="BeetleX.FastHttpApi" Version="1.4.9.8" />
```

Comment:
@IKende Same with BeetleX, please. We don't want a copy of the framework in our repo. It should be installed the way a real world user would install it.

Reply:
thank，all library has changed